packages/@react-native-windows/virtualized-list/src/VirtualizedList.js

```diff
@@ -34,7 +34,7 @@ import type {
   ViewToken,
   ViewabilityConfigCallbackPair,
 } from './ViewabilityHelper';
-import type {LayoutEvent} from '../Types/CoreEventTypes';
+import type {LayoutEvent} from 'react-native/Libraries/Types/CoreEventTypes';
```

Comment:
I think we use relative paths within the package.  - We should keep it relative, if nothing else they are less intensive on the bundler to work out the correct location.

Reply:
This was to fix a linting error, ` Cannot resolve module' ../Types/CoreEventTypes'. [cannot-resolve-module]`, I'm not sure of a way to make it relative while still passing the linter.

But we also import ScrollResponderType, ViewStyleProp and others in the same way. https://github.com/microsoft/react-native-windows/blob/827fa978fe5b706ca23b9beca5b9bcbd36dabb5d/packages/%40react-native-windows/virtualized-list/src/VirtualizedList.js#L30-L31

Reply:
Those examples are from another package, so they have to use a module level import.

Reply:
Maybe you need to modify .flowconfig?